radicle-httpd/src/api/v1/projects.rs

```diff
@@ -399,6 +407,89 @@ async fn issues_handler(
     Ok::<_, Error>(Json(issues))
 }
 
+#[derive(Debug, Deserialize, Serialize)]
+pub struct IssueCreate {
+    pub title: String,
+    pub description: String,
+    pub tags: Vec<Tag>,
+}
+
+/// Create a new issue.
+/// `POST /projects/:project/issues`
+async fn issue_creation_handler(
```

Comment:
```suggestion
async fn issue_create_handler(
```

Reply:
just saw this

![Screenshot from 2023-02-03 11-41-58](https://user-images.githubusercontent.com/14107758/216546802-542fc0c6-2e4d-4032-bc39-698edfb4c016.png)

we already have a `tree_handler_root` so maybe best if we stick to it, and use `issue_handler_create` here?

Reply:
I'm fine with both of them, but I think cloudheads suggestion sounds more natural in this case.

Reply:
kept @cloudhead suggestions, as it sounds more natural in 4cde666